frontend/src/component/TasksGallery.tsx

```diff
@@ -2,15 +2,20 @@ import {Task} from "../model/Task";
 import TasksCard from "./TasksCard";
 import "./TaskGallery.css";
 
-type Props={
-    tasks:Task[]
+type Props = {
+    tasks: Task[]
 }
-export default function TasksGallery(props:Props){
-    const taskCard=props.tasks.map((task)=>{
-        return <TasksCard key={task.id} task={task}/>
+export default function TasksGallery(props: Props) {
+    const taskCard = props.tasks.map((task) => {
```

Comment:
`taskCard` contains an array of Cards. Hence a name like `taskCards` would be more precise.